README.md

```diff
@@ -1,2 +1,157 @@
 # cesium-webpack-example
-The minimal recommended setup for an application using Cesium with Webpack.
+
+The minimal recommended setup for an application using [Cesium](https://cesiumjs.org/) with [Webpack](https://webpack.js.org/concepts/).
+
+[![Build Status](https://travis-ci.org/AnalyticalGraphicsInc/cesium-webpack-example.svg?branch=using-custom-loader)](https://travis-ci.org/AnalyticalGraphicsInc/cesium-webpack-example)
+
+### Running this application
+
+	npm install
+	npm start
+
+##### Available Scripts
+
+* `npm start` - Runs a webpack build with `webpack.config.js` and starts a development server
+* `npm run build` - Runs a webpack build with `webpack.config.js`
+* `npm run release` - Runs an optimized webpack build with `release.webpack.congif.js`
+
+### Requiring Cesium in your application
+ 
```

Comment:
We should provide an intro paragraph here with some context.  Having a bunch of different options with no explantation may confuse a lot of inexperienced developers.